### DISCUSSION TITLE
Claim SMR on Ledger

### DISCUSSION THREAD

**Discussion by michiprobsti** [iotaledger/firefly#7790] [Help & Support]
I recently downloaded the Firefly Shimmer Wallet - Version 2.1.10 - to claim my Shimmer rewards (from staking IOTA)

First I need to Choose Network -> I seelct **"Shimmer"**
Then I choose **"Claim Shimmer staking rewards"**
Then I choose **"Use Ledger Device"** + all its necessary steps.

After some time I see my staking rewards:

![image](https://github.com/iotaledger/firefly/assets/62829045/ae95e72a-5d8b-449f-9b6f-0a04009f489b)

After pressing **"Claim Rewards"** I need to verify my transaction. I have the following problem when trying to do so.

**I am concerned because Amount shows 0**

![image](https://github.com/iotaledger/firefly/assets/62829045/14e06824-6635-47da-9103-faba35908bec)

in addition to that my Ledger shows the following warning:
"_In order to claim SMR coins, you are now signing with IOTA private keys instead of SMR private keys. Are you really sure you want to proceed?_"

3 Questions:
* Should the wallet name be the same as in the Firefly app? Because it is just called 0 here.
* The amount should be displayed properly right? 
   * Or is this a "frontend bug"? (Wallet name is 0.... )
* The Warning _"...you are now signing with IOTA private keys instead of SMR private keys..."_ The message gets mentioned here (https://github.com/LedgerHQ/app-shimmer) but it is now a public archive so I am unsure if it is still up to date and valid.

How should I proceed? I do not want to risk losing my rewards.

Thanks in advance

**Comment by begonaalvarezd** [comment]
We will look into this asap and let you know if we need more information, we'll try to replicate the issue first, apologies for the troubles

**Reply by michiprobsti** [reply]
Are there any updates on this? 

I am still facing the issue in version 2.1.13

**Comment by michiprobsti** [comment]
Since there was no movement on this topic I decided to confirm the transaction even though the amount stated 0. 
Everything was working just fine. 

Either the amount just shows the "Wallet Name" (which seemed to be 0 in my case) or it just shows an invalid amount. Either way, I assume it is an UI bug.
